tests/auth_tests/test_forms.py

```diff
@@ -867,6 +869,31 @@ def test_bug_19349_bound_password_field(self):
         # value to render correctly
         self.assertEqual(form.initial["password"], form["password"].value())
 
+    @override_settings(ROOT_URLCONF="auth_tests.urls_admin")
+    def test_bug_34066_link_to_password_reset_in_helptext(self):
+        user = User.objects.get(username="testclient")
+        form = UserChangeForm(data={}, instance=user)
+        passsword_help_text = form.fields["password"].help_text
+        matches = re.search('<a href="(.*?)">', passsword_help_text)
+
+        # Get URL to UserChangeForm in admin via to_field (instead of pk)
+        admin_user_change_url = f"""{reverse(
+            "admin:%s_%s_change" % (user._meta.app_label, user._meta.model_name),
+            args=(user.username,),
+        )}?to_field=username"""
```

Comment:
On second thoughts creating a URL with to_field isn't required to test this issue – so the string interpolation can simply be removed:

```suggestion
        admin_user_change_url = reverse(
            "admin:%s_%s_change" % (user._meta.app_label, user._meta.model_name),
            args=(user.username,),
        )
```